projects/packages/analyzer/src/api/class-analyze-controller.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Automattic\Jetpack\Analyzer;
+
+class Analyze_Controller extends Controller {
+	public function get() {
+		try {
+			Locker::lock();
+		} catch ( \Throwable $th ) {
+			// TODO: extend default Exception class, and use instance_of instead.
+			if ( 'Locked already' === $th->getMessage() ) {
+				return $this->model->get_status();
+			}
+		}
+
+		$cmd = 'php ' . dirname( __DIR__ ) . '/../scripts/jp-analyze-parallel.php';
+
+		$descriptor_spec = array(
+			0 => array( 'pipe', 'r' ),
```

Comment:
Chances are you should close this pipe before returning. Or just do like `array( 'file', '/dev/null', 'r' )` in the first place.